packages/cosmic-swingset/src/anylogger-agd.js

```diff
@@ -4,15 +4,54 @@ import {
   getEnvironmentOption,
 } from '@endo/env-options';
 import anylogger from 'anylogger';
+import util from 'util';
 
-// Turn on debugging output with DEBUG=agoric
+//#region Golang zerolog-like output affordances
+let logFormat = 'plain';
+export const setLogFormat = format => {
+  logFormat = format;
+};
 
-const DEBUG_LIST = getEnvironmentOptionsList('DEBUG');
+let shortDateTimeFormatter;
+const plainTime = d => {
+  if (!shortDateTimeFormatter) {
+    shortDateTimeFormatter = new Intl.DateTimeFormat(undefined, {
+      timeStyle: 'short',
+    });
+  }
+  const shortTime = shortDateTimeFormatter.format(d).replace(/(\d) /, '$1');
+  return shortTime;
+};
+
+const levelToPlainLevel = {
+  error: 'ERR',
+  warn: 'WRN',
+  info: 'INF',
+  log: 'LOG',
+  debug: 'DBG',
+};
+
+/**
+ * @param {Date} d
+ * @returns {string}
+ */
+const jsonTime = d => {
+  const pad2 = n => (n < 10 ? `0${n}` : `${n}`);
+  const isoDate = `${d.getFullYear()}-${pad2(d.getMonth() + 1)}-${pad2(d.getDate())}`;
+  const isoTime = `${pad2(d.getHours())}:${pad2(d.getMinutes())}:${pad2(d.getSeconds())}`;
+  const tzo = d.getTimezoneOffset();
+  const isoTz = `${tzo < 0 ? '+' : '-'}${pad2(Math.floor(Math.abs(tzo) / 60))}:${pad2(Math.abs(tzo) % 60)}`;
+  return `${isoDate}T${isoTime}${isoTz}`;
+};
```

Comment:
I can't believe there is no way to do this in JS!

Reply:
Temporal fixes this, and there should probably be a comment to that effect. In the meantime, though, I do recommend something with less arithmetic:
```suggestion
/**
 * @param {Date} d
 * @returns {string}
 */
const dateToNumericOffsetString = d => {
  // TODO: Use Temporal.ZonedDateTime
  // `toString({ fractionalSecondDigits: 0 })`.`
  const pad2 = n => (n < 10 ? `0${n}` : `${n}`);
  const utcOffsetMinutes = -d.getTimezoneOffset();
  const utcOffsetAbsMinutes = Math.abs(utcOffsetMinutes);
  const utcOffsetString = `${utcOffsetMinutes < 0 ? '-' : '+'}${pad2(Math.floor(utcOffsetAbsMinutes / 60))}:${pad2(utcOffsetAbsMinutes % 60)}`;
  const utcDateWithSameLocalTime = new Date(d.getTime() + utcOffsetMinutes * 60e3);
  return utcDateWithSameLocalTime.toISOString().replace(/[.][0-9]+Z$/, utcOffsetString);
};
```

Reply:
Perfect within the constraints!  Thank you.